CMSgov/design-system
# [WNMGDS-2875] Release Script updates dependencies of child design systems now

## Summary

[Ticket](https://jira.cms.gov/projects/WNMGDS/2875)

- Main changes happened to the beginning of the bumpVersions function.  
- Now we manually update our package.json files
- Use npm version to update versions of items in workspaces as before
- Add getPackages function that pulls in our design system and examples dirs and package.json files.

## How to test

1. Run the `yarn release` script.
2. Confirm all of the appropriate json files are updated. One for each child design system, each example folder and the versions.json file.
3. Updated files:
[ ] - versions.json
[ ] - examples/astro/package.json
[ ] - examples/preact-app/package.json
[ ] - examples/preact-react-app/package.json
[ ] - examples/react-app/package.json
[ ] - packages/design-system/package.json
[ ] - packages/ds-cms-gov/package.json
[ ] - packages/ds-healthcare-gov/package.json
[ ] - packages/ds-medicare-gov/package.json

## Checklist
